docs/platforms/javascript/common/best-practices/micro-frontends.mdx

```diff
@@ -192,3 +192,17 @@ captureException(new Error("oh no!"), (scope) => {
   return scope;
 });
 ```
+
+## SDK Version Alignment
+
+Starting with version 8, if you have multiple Sentry JavaScript SDKs on the same page, they will [only interact with each other](https://github.com/getsentry/sentry-javascript/pull/12206) if they're all using the same version. This prevents unwanted cross-SDK interactions, for example, from browser extensions or 3rd party scripts.
+
+However, for micro frontends, where you _want_ SDK interaction between multiple SDKs, you'll need to ensure that all SDKs are using the same version.
```

Comment:
Perhaps something like: 

However, the micro frontend integration depends upon multiple configurations interacting with the same Sentry instance. This means that all microfrontends and the host _must_ use the same exact version of Sentry.

Reply:
> However, the micro frontend integration depends upon multiple configurations interacting with the same Sentry instance.

I don't think it strictly depends on same versions. Some MFE<>Sentry users just use Sentry in the host app and not in the MFEs at all. For them, the "multiple SDKs" part doesn't apply. However, I'll reword this because it's a bit misleading, I agree.